service/raft/group0_state_machine.cc

```diff
@@ -6,6 +6,7 @@
  * SPDX-License-Identifier: LicenseRef-ScyllaDB-Source-Available-1.0
  */
 #include "service/raft/group0_state_machine.hh"
```

Comment:
> raft: reload auth cache on snapshot application
> Receiving snaphot is a rare event so as a simplification
> we'll be reloading the whole cache instead of trying to merge
> states, especially that expected size is small, below 100 records.
> 
> Reloading is non-disruptive operation, old entries are removed
> only after all entries are loaded. If entry is updated, shared
> pointer will be atomically replaced in a cache map.

I think this is acceptable.

Please ask the cloud team to survey auth table row counts in our fleet.